pages/changelog.js

```diff
@@ -0,0 +1,92 @@
+import Page from "../components/Page";
+import PageHead from "../components/PageHead";
+import { CalendarIcon } from "@heroicons/react/20/solid";
+
+export default function Changelog() {
+  const colors = {
+    addition: "text-green-800 bg-green-100",
+    removal: "text-red-800 bg-red-100",
+  };
+  const changes = [
```

Comment:
Just an idea: We're generating the changelog.md file automatically already. Can't we do some magic in an endpoint with this markdown file? By doing so we could get rid of that manual step 🤔

Reply:
Good idea, but I think the goal of this page is to keep it super user friendly and high level, which I don't think is possible from the generated changelog